Libraries/Connect/Internal/Interceptors/ConnectInterceptor.swift

```diff
@@ -241,23 +241,28 @@ extension ConnectInterceptor: StreamInterceptor {
     }
 }
 
-private extension ProtocolClientConfig {
+extension ProtocolClientConfig {
```

Comment:
Likely necessary to test without a large refactor to how the public APIs leverage this.